src/ui_fsmenu/options.cc

```diff
@@ -301,6 +301,10 @@ Options::Options(MainMenu& fsmm, OptionsCtrl::OptionsStruct opt)
                      UI::PanelStyle::kFsMenu,
                      Vector2i::zero(),
                      _("Scroll when the mouse cursor is near the screen edge")),
+     invert_movement_(&box_ingame_,
+                     UI::PanelStyle::kFsMenu,
+                     Vector2i::zero(),
+                     _("Invert mouse movement via right click")),
```

Comment:
Sounds a bit unclear IMHO. How about something like
```suggestion
                     _("Invert click-and-drag map movement direction")),
```